articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+# How to build a static site with GatsbyJs
+![Hero image](/engineering-education/howto-build-astatic-site-with-gatsbyjs/heroimage.png)
+[*Image Source: Unsplash](https://unsplash.com/photos/uyfohHiTxho)
+
+When I first heard about GatsbyJS, I was confused about what it was, and how I would get started. As time went on, I learned more about GatsbyJS and found out it is awesome. My goal for this article is not to confuse you about GatsbyJS, but to help you get started with building your very first static site using GatsbyJS.
+
+GatsbyJS is used to build websites powered by [JAMStack](https://jamstack.org/what-is-jamstack/). JAMStack is a modern framework for creating websites and software. The “JAM” is an acronym for JavaScript, APIs, and HTML markup, JAMStack sites do not need a database, unlike websites designed using WordPress or Drupal. 
+In JAMStack architecture, the frontend and backend are [decoupled](https://www.gatsbyjs.com/docs/glossary/#decoupled). During the building process, Gatsby creates JavaScript, HTML, and CSS files used for the frontend, while the backend is a content API that can be a headless [CMS](https://en.wikipedia.org/wiki/Content_management_system), a hosted datastore, or a custom application that returns JSON or XML.
+[Matt Biilmann](https://www.netlify.com/oreilly-jamstack/) and [Chris Bach](https://www.linkedin.com/in/christianbachdk/) came up with the name "JAMStack" because they were creating modern web development workflows and capabilities at Netlify.
+
+### Advantages of JAMStack architecture
+- Speed: They load faster than sites using [monolithic architectures](https://whatis.techtarget.com/definition/monolithic-architecture#:~:text=A%20monolithic%20architecture%20is%20the,and%20unable%20to%20be%20changed.).
+- Flexibility for Hosting: Being a static file, JAMStack sites can be hosted anywhere. To get the best performance and security, it is preferable to use an object storage service and content delivery networks such as [Netlify](https://www.netlify.com/), [Render](https://www.gatsbyjs.com/docs/deploying-to-render/), or Amazon Web Services’ [S3 and CloudFront](https://www.gatsbyjs.com/docs/deploying-to-s3-cloudfront/).
+- Improved security: Because JAMStack sites don’t have software layers and databases, this makes them invulnerable to server-side code injection or SQL injection attacks.
```

Comment:
- If they don't have software layers and databases, what do they consist of then? Elaborate so your reader can follow.
- SQL injections _are_ a kind of server-side code attack. Rephrase to clarify, and remove extraneous words = terms have been confused

Reply:
Done